binance/async_client.py

```diff
@@ -1759,6 +1759,9 @@ async def futures_symbol_ticker(self, **params):
     async def futures_orderbook_ticker(self, **params):
         return await self._request_futures_api("get", "ticker/bookTicker", data=params)
 
+    async def futures_index_price_constituents(self, **params):
```

Comment:
@yzh-pelle we're missing assigning the .Doc here

Reply:
Do I need to add to the docstring?
:returns: API response

Or something else?